examples/conftest.py

```
@@ -18,3 +18,18 @@
     pass
 else:
     np.random.seed(0)
+
+
+def pytest_terminal_summary(terminalreporter, exitstatus, config):
```

Ruff (0.14.0)

Unused function argument: `config`

(ARG001)